lib/Aggregate.js

```diff
@@ -313,4 +313,10 @@ Aggregate.load = async function({version, time, ...constructorParams} = {}) {
   return aggregate
 }
 
+Aggregate.getState = async function(...args) {
+  const {state} = await this.load(...args)
```

Comment:
do we want to raise error if object can't be found?

Reply:
Shouldn't load do that?

Reply:
Aggregate don't have a not found scenario, but I added .getState() to AggregateWithKey to handle its null scenario